src/jit-format/jit-format.cs

```diff
@@ -555,7 +556,7 @@ public static string rewriteCompileCommands (string compileCommandFile, string p
                             {
                                 compileCommand = compileCommand + " " + option;
                             }
-                            else if (option.Contains("src/jit"))
+                            else if (option.Contains("src/jit") && !option.StartsWith("/Fp"))
```

Comment:
Could you please add a comment about why we are ingoring "/Fp" here so it won't confuse us in the future?

Reply:
I added a comment. Thank you for the review.